### PR TITLE
fix for coingecko endpoint prices

### PR DESCRIPTION
**Pull request by poocart**
Fixes CoinGecko token price parsing for multiple matching token symbols. Initial approach used fetch by symbols, this approach replaces prices endpoint to fetch by token contract addresses.

### CODE REVIEW

**Review by graszka22 — COMMENTED** [review]
It seems to me it doesn't work for ETH. It shows just 0$ all the time even though it shows I have a small amount of eth. And as far as I know ETH doesn't have a real contract, right? So maybe that's it?

**Comment by poocart** [comment]
Interesting, last night I checked it seemed to be good, but indeed it's not. After import was able to get 0 price as well. Updated this, thanks 👍

File: src/services/assets.js [hunk 1]
```diff
@@ -382,7 +384,13 @@ export async function getExchangeRates(assetSymbols: string[]): Promise<?Object>
...
+    : await getCoinGeckoTokenPrices(assets);
+
+  // ether price fetched separately as it doesn't fit into CoinGecko token price endpoint
+  const etherPrice = await getCoinGeckoEtherPrice();
+  if (etherPrice) {
+    rates = { ...rates, [ETH]: etherPrice };
+  }
 
   // by any mean if CoinGecko failed let's try legacy way
   if (!useLegacyCryptoCompare && isEmpty(rates)) {
```
Comment (graszka22) [inline src/services/assets.js:396]: if rates is empty before fetching eth, this check won't detect it

**Review by graszka22 — APPROVED** [review]
now it works for me 👍 one suggestion below

**Pull request merged**